### PR TITLE
Install netifaces package in sonic-slave docker and sonic image

### PR DESCRIPTION
**Pull request by nikos-github**
Change required for https://github.com/Azure/sonic-utilities/pull/194

### CODE REVIEW

**Comment by lguohan** [comment]
better to rename the commit message to install netifaces package in sonic-slave docker.

**Comment by nikos-li** [comment]
@lguohan Please do not merge this yet.

**Comment by nikos-li** [comment]
@lguohan Please review this again.

File: build_debian.sh [hunk 1]
```diff
@@ -300,6 +300,11 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT easy_install pip
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'docker-py==1.6.0'
 ## Note: keep pip installed for maintainance purpose
 
+## Get gcc and python dev pkgs
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libpython2.7-dev
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install gcc
```
Comment (jleveque) [inline build_debian.sh]: Unless you have a reason to keep them separate, you can combine lines 304 and 305 into one command, e.g.:

`sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install gcc libpython2.7-dev`

File: build_debian.sh [hunk 2]
```diff
@@ -342,6 +347,10 @@ if [ "${enable_organization_extensions}" = "y" ]; then
    fi
 fi
 
+## Remove gcc and python dev pkgs
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove libpython2.7-dev
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove gcc
```
Comment (jleveque) [inline build_debian.sh]: Unless you have a reason to keep them separate, you can combine lines 351 and 352 into one command, e.g.:

`sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y remove gcc libpython2.7-dev`

**Comment by jleveque** [comment]
Retest this please.

File: build_debian.sh (hunk 1 above)
Comment (nikos-li) [reply build_debian.sh]: Sure will do that.

File: build_debian.sh (hunk 2 above)
Comment (nikos-li) [reply build_debian.sh]: Sure will do that.

File: build_debian.sh [hunk 3]
```diff
@@ -300,6 +300,10 @@ sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT easy_install pip
 sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'docker-py==1.6.0'
 ## Note: keep pip installed for maintainance purpose
 
+## Get gcc and python dev pkgs
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install gcc libpython2.7-dev
+sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'netifaces==0.10.6'
```
Comment (lguohan) [inline build_debian.sh]: sonic-utilities package is also installed in docker-sonic-vs, so can you also install the netifaces in the docker vs. You need to add this in platform/vs/docker-sonic-vs/Dockerfile.j2.

Comment (nikos-github) [reply build_debian.sh]: Not needed. netifaces is in docker-sonic-vs

**Comment by nikos-github** [comment]
@lguohan Comments addressed.

**Pull request merged**